hydromt_wflow/wflow.py

```diff
@@ -4527,10 +4544,10 @@ def setup_irrigation(
         lai_threshold: float = 0.2,
         lulcmap_name: str = "meta_landuse",
         output_names: Dict = {
-            "land~irrigated-paddy_area__count": "paddy_irrigation_areas",
-            "land~irrigated-non-paddy_area__count": "nonpaddy_irrigation_areas",
-            "land~irrigated-paddy__irrigation_trigger_flag": "paddy_irrigation_trigger",
-            "land~irrigated-non-paddy__irrigation_trigger_flag": "nonpaddy_irrigation_trigger",  # noqa: E501
+            "land~irrigated-paddy_area__number": "demand_paddy_irrigated_mask",
+            "land~irrigated-non-paddy_area__number": "demand_nonpaddy_irrigated_mask",
```

Comment:
Merge conflicts issues there (wflow variables were updated in another PR and should be kept)